src/carts/PaymentMapper.js

```diff
@@ -61,6 +62,7 @@ class PaymentMapper {
      */
     _fillPayment(payment, partialCart) {
         partialCart.payment = payment;
+        partialCart.payments = [ payment ];
```

Comment:
Can Magento only have one payment?

Reply:
@cjelger , I've tested that if I post two payment methods via our API one after the other the second one replaces the first. The returned cart has always one payment method. So, I assume that current we support one payment pert cart.

Reply:
@LSantha what if we add the payment through console or any other flow.